### PR TITLE
fix brackets, add user/project endpoints

### PR DESCRIPTION
**Pull request by henderkes**

### CODE REVIEW

File: backend/api/search.py [hunk 1]
```diff
@@ -128,6 +128,101 @@ def search_api():
...
+            'wildcard': {'title': '*' + title + '*'}
+        })
+
+    try:
+        res = g.es.search(index="knexdb", body=request_json)['hits']['total']
+        return jsonify(res)
+    except Exception as err:
+        raise ApiException(str(err), 400)
+
+
```
Comment (Drakulix) [inline backend/api/search.py:156]: This amount of duplicated code is hideous as fuck.
Please make a new `es_query` function or something like that out of it, where you just stuff in `query` and get the final `request_json`.

**Review by Drakulix — CHANGES_REQUESTED** [review]
Ok. Except for that search/count call.

File: backend/api/search.py (hunk 1 above)
Comment (mharasic) [reply backend/api/search.py:156]: :+1:

Comment (henderkes) [reply backend/api/search.py:156]: should be fixed already

Comment (Drakulix) [reply backend/api/search.py:156]: nice

**Review by Drakulix — DISMISSED** [review]
:+1:

**Comment by henderkes** [comment]
don't merge yet, testing currently

**Comment by henderkes** [comment]
endpoints tested, ready for merge

**Review by Drakulix — APPROVED** [review]

**Pull request merged**